addon/ng2/utilities/get-dependent-files.ts

```diff
@@ -0,0 +1,105 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import * as ts from 'typescript';
+
+interface MapType {
```

Comment:
In order to facilitate readability where it's used, the type name should reflect its content (the same reason you don't name your variables with generic names), so yes I think it's best if the type has the name of how it will be used :)